### PR TITLE
Reduce bundle size (7.2MB -> 5.4MB) with consistent grommet imports

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

**Comment by csillag** [comment]
Considering the current build system, does using "import type" (instead of simple import) make any difference in the compiled product? I see a mixed practice in the current codebase, including this PR.

**Review by csillag — APPROVED** [review]
I see no issues, besides the small thing of inconsistent usage of `import type`.

**Comment by lukaw3d** [comment]
Ah yeah, I'll remove that import type (has nothing to do with reducing size)

File: internals/jest/jest.base-config.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const config = {
   transform: {
     '^.+\\.(js|jsx|mjs|cjs|ts|tsx)$': '<rootDir>/internals/jest/babelTransform.js',
   },
-  transformIgnorePatterns: ['/node_modules/(?!(cborg)/)'],
+  transformIgnorePatterns: ['/node_modules/(?!(cborg)|(grommet/es6)|(grommet-icons/es6)/)'],
```
Comment (buberdds) [inline internals/jest/jest.base-config.js:21]: I would go with one capturing group
`transformIgnorePatterns: ['/node_modules/(?!(cborg|grommet/es6|grommet-icons/es6)/)'],`

**Review by buberdds — APPROVED** [review]

File: internals/jest/jest.base-config.js (hunk 1 above)
Comment (lukaw3d) [reply internals/jest/jest.base-config.js:21]: yep yep

**Pull request merged**